.gitattributes

```diff
@@ -0,0 +1,11 @@
+# inspired by https://github.com/composer/composer/blob/2406a094d3b2e5d73ce105764b36898ebd19b69d/.gitattributes
+
+# Auto-detect text files, ensure they use LF.
+*       text=auto eol=lf
+
+# These files are always considered text and should use LF.
+# See core.whitespace @ https://git-scm.com/docs/git-config for whitespace flags.
+*.php   text eol=lf whitespace=blank-at-eol,blank-at-eof,space-before-tab,tab-in-indent,tabwidth=4 diff=php
+*.json  text eol=lf whitespace=blank-at-eol,blank-at-eof,space-before-tab,tab-in-indent,tabwidth=4
+*.test  text eol=lf whitespace=blank-at-eol,blank-at-eof,space-before-tab,tab-in-indent,tabwidth=4
+*.yml   text eol=lf whitespace=blank-at-eol,blank-at-eof,space-before-tab,tab-in-indent,tabwidth=2
```

Comment:
Unsere redaxo-yml-dateien formatieren wir mit 4 spaces. weiß nicht, ob hier dann am ende ne 4 stehen sollte.
(in den actions sind wir uneinheitlich, teilweise haben wir da 2 spaces)